aspnetcore/mvc/advanced/custom-model-binding/sample/CustomModelBindingSample/Startup.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
```

Comment:
Now you've got a super simple startup that shows everything you need. I really like how simply you used in-mem db and seeded it.